curriculum/challenges/english/blocks/lab-game-character-stats/68cabc534d2e741f33a3fe1d.md

```diff
@@ -0,0 +1,536 @@
+---
+id: 68cabc534d2e741f33a3fe1d
+title: Build a Game Character Stats Tracker
+challengeType: 27
+dashedName: lab-game-character-stats
+---
+
+# --description--
+
+In this lab, you'll build a game character stats tracker. The program will allow you to create a character with specific attributes, update those attributes, and retrieve the current stats of the character.
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
+
+**User Stories:**
+
+1. Create a class named `GameCharacter` that represents a game character and manages character stats.
+2. When instantiated, a new `GameCharacter` object should take a `name` parameter and initialize the following attributes:
```

Comment:
It's not correct to say that an "object should take a parameter". Let's remove the mention to `name` and let campers name it whatever they want, which was why users stories have been written like this.
```suggestion
2. When instantiated, a new `GameCharacter` object should have the following attributes:
```